src/DynamoCoreWpf/UI/Themes/Modern/DynamoModern.xaml

```diff
@@ -2061,38 +2086,34 @@
             <Style x:Key="DecreaseButton"
                    BasedOn="{x:Null}"
                    TargetType="{x:Type RepeatButton}">
-                <Setter Property="MinWidth" Value="10" />
+                <Setter Property="MinWidth" Value="12" />
                 <Setter Property="Template">
                     <Setter.Value>
                         <ControlTemplate TargetType="{x:Type RepeatButton}">
                             <Grid>
                                 <Border Name="Border"
                                         Margin="0"
                                         Padding="0"
-                                        Background="#3C3C3C" />
-                                <ContentPresenter x:Name="ContentPresenter"
-                                                  HorizontalAlignment="Center"
-                                                  VerticalAlignment="Center"
-                                                  Content="{TemplateBinding Content}"
-                                                  ContentTemplate="{TemplateBinding ContentTemplate}"
-                                                  ContentTemplateSelector="{TemplateBinding ContentTemplateSelector}" />
+                                        Background="{TemplateBinding Background}">
+                                    <Path Margin="1,0,0,1"
+                                          HorizontalAlignment="Center"
+                                          VerticalAlignment="Center"
+                                          Data="M-2.5,-1 L0,2 L2.5,-1"
+                                          Fill="{x:Null}"
```

Comment:
same comment here

Reply:
Seems to work fine without; have removed.